Formula/gst-plugins-bad.rb

```diff
@@ -38,14 +38,14 @@ class GstPluginsBad < Formula
   depends_on "rtmpdump" => :optional
   depends_on "schroedinger" => :optional
   depends_on "sound-touch" => :optional
-  depends_on "srtp" => :optional
+  depends_on "srtp@1.5" => :optional
   depends_on "libvo-aacenc" => :optional
 
   def install
     args = %W[
       --prefix=#{prefix}
       --disable-yadif
-      --disable-sdl
```

Comment:
Didn't look at the commit messages (which are usually empty on Homebrew). Thanks for explaining. This cleanup is OK.